libs/langchain/langchain/chat_models/huggingface_chat_wrapper.py

```diff
@@ -0,0 +1,165 @@
+from typing import Any, List, Optional, Union
+
+from huggingface_hub import list_inference_endpoints
+from transformers import AutoTokenizer
```

Comment:
since these are optional dependencies they should only be imported inside functions as needed (instead of being imported at top level)

Reply:
Thanks, it looks like this was causing the test failures!